use-host-from-sdk/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+rm -rf TestDir
+mkdir TestDir
+pushd TestDir
+
+# Verify that the host from the SDK is usable and nuget is not needed
+# for building a HelloWorld application
+
+cat > nuget.config <<EOF
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+  </packageSources>
+</configuration>
+EOF
+
+dotnet new console
+dotnet run --no-restore
```

Comment:
nit: it may not be obvious for someone reading this test where/how this is using the sdk host.

Reply:
Any suggestions on how to make this more obvoius? Do you think the test should explicitly tests for the host file here? Or is it more about explicitly documenting something?

Reply:
Any thoughts, @tmds ?

Reply:
You could explicitly use the apphost to launch the app:
```
dotnet build --no-restore
bin/Debug/netcoreapp*/console
```